src/sentry/plugins/sentry_mail/activity/base.py

```diff
@@ -138,7 +138,8 @@ def get_headers(self):
         group = self.group
 
         headers = {
-            'X-Sentry-Team': project.team.slug,
+            # TODO(jess): is this ok to just delete? or should we replace with org or something?
+            # 'X-Sentry-Team': project.team.slug,
```

Comment:
i'm not sure what the implications are of just removing these

Reply:
@tkaemming